Gemfile.lock

```diff
@@ -69,48 +70,50 @@ GEM
     ast (2.3.0)
     attr_encrypted (3.0.3)
       encryptor (~> 3.0.0)
-    awrence (0.1.0)
-    aws-sdk (2.6.12)
-      aws-sdk-resources (= 2.6.12)
-    aws-sdk-core (2.6.12)
+    awrence (1.0.0)
+    aws-sdk (2.10.21)
```

Comment:
I wouldn't worry about this dependency-of-a-dependency too much. Likely just ` carrierwave-aws` and `aws-sdk-resources` (used by Shrine, I believe)